tests/behat/theme_boost_union_block_region.feature

```diff
@@ -0,0 +1,553 @@
+@theme @theme_boost_union  @theme_boost_union_block_config @javascript
```

Comment:
If the block regions settings are moved to the "Feel" settings page, this feature file should be renamed to a "Feel -> Blocks" feature.